cmd/vic-machine/common/limits.go

```diff
@@ -30,6 +30,8 @@ type ResourceLimits struct {
 	VCHMemoryReservationsMB *int              `cmd:"memory-reservation"`
 	VCHMemoryShares         *types.SharesInfo `cmd:"memory-shares"`
 
+	StorageQuotaGB *int `cmd:"storage-quota"`
```

Comment:
storage-quota is for both images and container vms. A better name could be, but the current name is fine I think, according to the epic description.

Reply:
There is definite scope for confusion here, and it's hard to change the name of an option after release.

I think it comes down to the semantics of this _specific_ quota option - should the usage calculation include the usage in volume stores, which may also have their own specific quotas? If yes, then we leave it as it is as it's the global quota.

It's not enforced as a global quota at the moment but it's also not an image-store-quota because it encompasses containers as well - this implies that we may need/want `image-store-quota`, `container-store-quota`, and `volume-store-quota` granular options at a later date.

An alternative would be to allow expressing quota per vsphere datastore, which is, in most cases, likely to be what the viadmin actually wants to express. Given that interpretation then `--storage-quota` without an explicit datastore part to the value defaults to the datastore used by image-store, which also happens to be the only option for container-store at this time.

Summary: this is going to be confusing no matter our choice until we have added quota support for the remaining components.

Reply:
Since we have help information in command line, I am fine with storage-quota for image/containers, and volume-storage-quota for all the volumes usage, or specific format definition for each volume such as volume-name:quota.